Google.Api.Gax.Grpc/Rest/TranscodingOutput.cs

```diff
@@ -0,0 +1,75 @@
+﻿/*
+ * Copyright 2022 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+
+namespace Google.Api.Gax.Grpc.Rest;
+
+/// <summary>
+/// The result of transcoding a protobuf request using an HttpRule.
+/// This is produced by <see cref="HttpRuleTranscoder"/>.
+/// </summary>
+internal sealed class TranscodingOutput
```

Comment:
nit: Maybe `TranscodedRequest` and then `CreateRequest` => `CreateHttpRequest`

Reply:
Maybe later - we tend to only really think about transcoding as the proto->HTTP side of things, so I'm not worried about being ambiguous there. CreateRequest => CreateHttpRequest (or ToHttpRequestMessage) makes sense.